debian/homestead.init.d

```diff
@@ -228,6 +245,17 @@ do_stop()
 }
 
 #
+# Function that runs the daemon/service in the foreground
+#
+do_run()
+{
```

Comment:
As discussed, this is a bit difficult/messy because "run" services don't write a PID.  We agreed not to do this.